platform/go/go-std/config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "target": {
+    "path": "/update?queries=10"
```

Comment:
I don't recall playing with queries > 10 for this app. I looked at the [test spec](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#database-updates) and the typical values are 1~20 -- and indeed it was way off when we [mistakenly went with a bogus 100 here](https://github.com/getsentry/sentry-sdk-benchmark/commit/6f9be0df1e9d01003621861058eb8bed5018949e).

We can probably settle on 10, and let `target.path` convey the path and query string.